### ISSUE TITLE
LAPACK make fails: “recipe for target 'znep.out' failed” error

### ISSUE THREAD

**Issue by canberkgurel** [Reference-LAPACK/lapack#341]
I created a `make.inc` file using the example provided. 

When I tried to install with `make` I received the following error:

```
make[2]: Leaving directory '/home/csgurel/svo_install_overlay_ws/src/lapack/TESTING/LIN'
Testing COMPLEX16 LAPACK linear equation routines
./LIN/xlintstz < ztest.in > ztest.out 2>&1
SEP: Testing Symmetric Eigenvalue Problem routines
./EIG/xeigtstz < sep.in > zsep.out 2>&1
Makefile:467: recipe for target 'zsep.out' failed
make[1]: *** [zsep.out] Error 139
make[1]: Leaving directory '/home/csgurel/svo_install_overlay_ws/src/lapack/TESTING'
Makefile:54: recipe for target 'lapack_testing' failed
make: *** [lapack_testing] Error 2
```

As stated [here](https://unix.stackexchange.com/questions/428394/lapack-make-fails-recipe-for-target-znep-out-failed-error?newreg=381839326d7c4f979798f010ad34ba23) 

I tried the suggested solution (`ulimit -s unlimited`) but it didn't solve this error. Can anyone suggest a solution?

**Comment by zhouxinxin121** [comment]
I have the same  error,but `ulimit -s 100000` solve my error.

**Comment by Hadizs** [comment]
Hi, Did you solve this problem?

**Comment by martin-frbg** [comment]
In case he hasn't, can you provide more details about what system you are trying to compile on, and what your make.inc looks like ? This is essentially a crash of one of the tests, and it is usually caused by the test code allocating some big array of test data statically, i.e. on the stack. The usual default stack size on Linux is 8192k, on other systems it may be even smaller - and the array is big enough to cause an overflow at that limit.
Depending on your system, there may be a "hard" limit for the stacksize imposed that you cannot override as a user (or if you have Java code in the mix e.g. from some IDE that uses Java, you may need other means to change the stacksize), or you
may be seeing a completely different problem than the usual "segmentation fault" that a stack overflow would result in.

**Comment by langou** [comment]
Closed issue via #492
